src/openforms/scss/admin/_app_overrides.scss

```
@@ -98,6 +98,31 @@ $djai-padding--tablet: 30px;
   padding-top: 1em;
 }
 
+.admin-login-divider {
+  text-align: center;
+  clear: both;
+  padding-top: 1em;
+
+  width: 100%;
+  text-align: center;
+  border-bottom: 1px solid #ccc;
+  line-height: 0.1em;
+  margin: 10px 0 10px;
+
+  span {
+    background: #fff;
+    padding: 0 10px;
+  }
+}
```

Alternative:

```suggestion
.admin-login-divider {
  display: flex;
  flex-direction: row;
  align-items: center;
  padding-block-start: 1em;

  &:after,
  &:before {
    content: '';
    display: block;
    border-bottom: 1px solid #ccc;
    flex-grow: 1;
  }

  > span {
    flex-shrink: 0;
    padding-inline: 10px;
  }
}
```

the flexbox makes this automatically centered without need for magic numbers, and you don't need the background color to create an overlay so it works in whatever theme you have (just need to fix the color reference for the border color).

---

I'm not sure if I should add a new variable for the border color, or if I can use `$color_dark_light`?